### PR TITLE
Vignettes contacts

### PR DESCRIPTION
**Pull request by jakubcervenka**
just changing two instances of jacob@.... to stresstest@...
I have not touched the DESCRIPTION coz I don't know how to edit but not as important probably

### CODE REVIEW

**Review by jacobvjk — COMMENTED** [review]
lgtm, but @MirijaHa please check if you are fine with it too

**Review by MirijaHa — APPROVED** [review]

**Comment by MirijaHa** [comment]
suggestion: before jacob leaves we have to update the contact author in the desciption. Could also do that now, simply exchange given and email with 2dii info and make sure to leave roles as they are, e.g. as done here: https://github.com/2DegreesInvesting/r2dii.analysis/blob/96d3965d68e1dedbd9e4ddcfca57857dc8590c67/DESCRIPTION#L19

**Review by jacobvjk — APPROVED** [review]
lgtm

File: DESCRIPTION [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 Version: 0.0.0.9003
 Authors@R: c(
-    person("Jacob", "Kastl", , "jacob@2degrees-investing.org", role = c("aut", "cre")),
-    person("2 Degrees Investing Initiative", , , "contact@2degrees-investing.org", role = c("cph", "fnd"))
-  )
+    person(given = "Jacob",
+           family = "Kastl",
+           role = c("aut", "cre"),
+           email = "jacob.kastl@gmail.com"),
+    person(given = "Mirja",
```
Comment (MirijaHa) [inline DESCRIPTION:10]: please remove me from the authors section, otherwise LGTM

**Review by MirijaHa — CHANGES_REQUESTED** [review]

**Review by MirijaHa — APPROVED** [review]

**Pull request merged**